src/Sylius/Behat/Context/Api/Admin/ManagingShippingMethodsContext.php

```diff
@@ -205,6 +205,19 @@ public function theShippingMethodShouldAppearInTheRegistry(string $shippingMetho
         );
     }
 
+    /**
+     * @Given /^(this shipping method) should still be in the registry$/
+     */
+    public function thisShippingMethodShouldAppearInTheRegistry(ShippingMethodInterface $shippingMethod): void
+    {
+        $shippingMethodName = $shippingMethod->getName();
```

Comment:
```suggestion
        $name = $shippingMethod->getName();
```
as it is shipping methods context